### ISSUE TITLE
null pointer deref after removing interface

### ISSUE THREAD

**Issue by reusee** [tailscale/tailscale#532]
**Describe the bug**
```
2020/07/07 16:57:32 6.2M/18.9M monitor: netlink msg {Length:96 Type:unknown(21) Flags:0 Sequence:0 PID:0}, "\x02\x18\x80\x00<\x00\x00\x00\b\x00\x01\x00\n\x00\x00\x01\b\x00\x02\x00\n\x00\x00\x01\b\x00\x04\x00\n\x00\x00\xff\x12\x00\x03\x00L2-10.0.0.1-0\x00\x00\x00\b\x00\b\x00\x80\x00\x00\x00\x14\x00\x06\x00\xff\xff\xff\xff\xff\xff\xff\xff v)\x00 v)\x00"
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x10 pc=0x4372cc]

goroutine 49 [running]:
tailscale.com/wgengine/monitor.(*Mon).pump(0x10ac140)
	/Users/reus/github/tailscale/wgengine/monitor/monitor.go:117 +0x78
created by tailscale.com/wgengine/monitor.(*Mon).Start.func1
	/Users/reus/github/tailscale/wgengine/monitor/monitor.go:79 +0x5c
```

**To Reproduce**
1. start a vpn program which will create a tap device
2. start tailscaled
3. stop the vpn program and the tap device will be removed
4. crash 

**Expected behavior**
no crash

**Version information:**
os: arm linux
tailscaled: master branch

**Comment by bradfitz** [comment]
Sigh. I introduced that just yesterday.

You're having all sorts of good (or bad) luck. Thanks for testing unstable! It's very helpful.

**Comment by bradfitz** [comment]
Fixed in latest unstable build (v0.100.0-9).
